### ISSUE TITLE
Making Glass More Glass

### ISSUE THREAD

**Issue by lzlrd** [Athemis/PixelPerfectionCE#5]
After using this resource pack for months, I can't help but think that the "wooden bars" in the middle of glass blocks and panes are quite intrusive. They also ruin reflective effects of shaders.

**Comment by Athemis** [comment]
I agree. That bothered me as well. I'll see if I can come up with something better. Perhaps even reverting to the old style in PP 3.6.
